bots/create-replication-bots/index.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const registerReplicationBots = require("./register-replication-bots");
+const sendCustomResourceResponse = require('../../lib/sendCustomResourceResponse');
+const logger = require('leo-logger');
+
+exports.handler = (event, _, callback) => {
+	logger.log(JSON.stringify(event, null, 2));
+	try {
+		event.PhysicalResourceId = event.LogicalResourceId;
```

Comment:
@zirkerc There was an error about the Physical ID. I saw this was being done in the install bot so I copied it... I am not sure why this is needed. Can you explain?

Reply:
Custom resource need to create a physical Resource for the Logical one.  eg. Logical Dynamodb table `Stats` is actually `MyStack-Stats-ADCE234`. Cloudformation is expecting you to export the PhysicalResourceId.  Because it is just an entry in the Cron table we just use the LogicalResourceId as the PhysicalResourceId